docs/conf.py

```diff
@@ -22,153 +22,164 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+# sys.path.insert(0, os.path.abspath('.'))
 
-BOREALISPATH = os.path.abspath('..')
-os.environ['BOREALISPATH'] = BOREALISPATH
+BOREALISPATH = os.path.abspath("..")
+os.environ["BOREALISPATH"] = BOREALISPATH
 sys.path.insert(0, BOREALISPATH)
-sys.path.insert(1, f'{BOREALISPATH}/src')
-sys.path.insert(2, f'{BOREALISPATH}/src/experiment_prototype')
-sys.path.insert(3, f'{BOREALISPATH}/src/utils')
-sys.path.insert(4, f'{BOREALISPATH}/scheduler')
-sys.path.insert(5, os.environ['PATH'])
+sys.path.insert(1, f"{BOREALISPATH}/src")
+sys.path.insert(2, f"{BOREALISPATH}/src/experiment_prototype")
+sys.path.insert(3, f"{BOREALISPATH}/src/utils")
+sys.path.insert(4, f"{BOREALISPATH}/scheduler")
+sys.path.insert(5, os.environ["PATH"])
 
 # -- Pre-build configuration ----------------------------------------------
 
-# Doxygen: Reads all c++ source and header files, and parses the documentation to xml files 
-# for further reading. 
-run('doxygen')
+# Doxygen: Reads all c++ source and header files, and parses the documentation to xml files
+# for further reading.
+run("doxygen")
 
 # Breathe: parses xml files produced by doxygen and creates rst files for use by Sphinx Files
 # are re-generated each call, no table of contents file is created, and only rst files are
 # created. For more information, `breathe-apidoc --help`
 cur_dir = os.path.abspath(os.path.dirname(__file__))
-run(['breathe-apidoc', '--force', '--no-toc', '--generate', 'file', '-o', f'{cur_dir}/source', f'{cur_dir}/xml/'])
+run(
+    [
+        "breathe-apidoc",
+        "--force",
+        "--no-toc",
+        "--generate",
+        "file",
+        "-o",
+        f"{cur_dir}/source",
+        f"{cur_dir}/xml/",
+    ]
+)
 
 # Update the experiment subrepo so experiment files can be read into documentation
-run(['git', 'submodule', 'update', '--init'])
+run(["git", "submodule", "update", "--init"])
```

Comment:
Why is this one autoformatted different from the run command above?

Reply:
The one above was reformatted since it exceeded the line length